packages/main/src/Card.js

```diff
@@ -82,6 +84,26 @@ const metadata = {
 			type: URI,
 			defaultValue: null,
 		},
+
+		_headerActive: {
```

Comment:
activeHeader?

Reply:
This one is for the pressed state, but there could be a public property determining wether the header is active or not (activeHeader as you said) and then this would be renamed to _pressedHeader or similar.